.gitignore

```diff
@@ -30,3 +30,5 @@ coverage.txt
 
 # Exclude build directory
 build
+
+test/bdd/db
```

Comment:
you might also need to have codecov ignore test directory.

Reply:
ok